ray_julia_jll/deps/build_jll.jl

```diff
@@ -32,11 +32,3 @@ end
 if !isfile(joinpath(artifact_dir, library_name))
     error("Failed to build library: $(joinpath(artifact_dir, library_name))")
 end
-
-# Add entry to depot specific Overrides.toml
-overrides_toml = joinpath(first(DEPOT_PATH), "artifacts", "Overrides.toml")
-overrides_dict = isfile(overrides_toml) ? TOML.parsefile(overrides_toml) : Dict{String,Any}()
-overrides_dict[pkg_uuid] = Dict("ray_julia" => abspath(artifact_dir))
-open(overrides_toml, "w") do io
-    TOML.print(io, overrides_dict)
-end
```

Comment:
Users building locally will still want this

Reply:
yeah we need to remove it though when building the artifacts otherwise they won't get used... I think it can be a manual step that we can document